pydra/engine/submitter.py

```diff
@@ -1,14 +1,7 @@
 """Handle execution backends."""
 import asyncio
-import time
 from uuid import uuid4
-from .workers import (
-    SerialWorker,
-    ConcurrentFuturesWorker,
-    SlurmWorker,
-    DaskWorker,
-    SGEWorker,
-)
+from .workers import WORKERS
 from .core import is_workflow
 from .helpers import get_open_loop, load_and_run_async
```

Comment:
given that these functions are only used here, it might help with readability just having them here.

Reply:
agree